content/authoring/guidelines/reference-solution.md

```diff
@@ -7,15 +7,8 @@ next: /authoring/guidelines/translation/
 
-The reference solution snippet, created by the kata author or translator, has a very important role in the kata maintenance process. To help keep kata quality good, it should conform to a set of guidelines presented below. 
-
-This article contains a set of guidelines for kata authors and translators to create good code snippets for their kata. They were collected to help ensure that kata and translations are of sufficient quality and kata maintenance will be as easy as possible.
-
-## General Guidelines
-
-- **Conform to [General Coding Guidelines][authoring-guidelines-general-coding]**{id="general"}: the reference solution is code too, and, as such, should keep up to code quality standards. It will make maintenance of the kata much easier, and the readability of the reference solution can be very helpful for future maintainers and translators. The reference solution is not for you to write, but for others to read.
-- When creating a new kata, especially a performance oriented one, the reference solution should reflect **the minimal solution intended by the kata author to pass**{id="minimal-ref"}. Sometimes maintainers and translators are not sure what the author's original intention behind the kata was in terms of its difficulty, edge cases and additional restrictions. The reference solution of the initial language version can then be used as a baseline for other translations.
-- For reasons mentioned above, when a kata is translated, **the meaning of the reference solution from the initial version should be preserved**{id="keep-reqs"}. It's especially important for performance oriented kata. The reference solution of a translation should not significantly stand out from the original one. They do not have to be totally equivalent, and they can use different techniques, but they should correspond in terms of performance, handling of edge cases and invalid inputs, etc. The reference solution of a translation should not be significantly better or worse than the original one.
+The reference solution snippet, created by the kata author or translator, is a snippet which might appear as not very relevant, but when prepared incorrectly, it still can incur some additional, unnecessary maintenance cost. It might appear that the reference solution is just a solution like every other submitted by a user, and that the only important thing is to pass the tests. However, it's something more: the reference solution is run every time a kata or translation is republished, and it should be prepared in a way which would prevent the publication from failing in an unexpected way.
 
+For the above reasons, this snippet **should conform to [General Coding Guidelines][authoring-guidelines-general-coding]**: the reference solution is code too, and, as such, should keep up to code quality standards. It will make maintenance of the kata much easier, and the readability of the reference solution can be very helpful for future maintainers and translators.
```

Comment:
Isn't this too wordy? It's difficult to understand because of that and the passive tone.

```suggestion
The reference solution **should conform to [General Coding Guidelines][authoring-guidelines-general-coding]** because it must be maintained, unlike other submitted solutions. Always think about the future maintainers and translators.
```

I'm not sure having a separate page for this is worth it, but I do hate fixing **inelegant** one-liners that the translator was once proud of before disappearing.

Reply:
(note: the smiley is about the comment below the suggestion)

What is the "reference solution" should be told first, I believe. Currently, this cannot be deduced from the text. For instance, when I hear "ref solutino", I always think about the one used for the random tests, in the tests suite.
Using kazk suggestion:

```suggestion
The reference solution is the solution of the author/translator that is showing up in the kata solutions page. It **should conform to [General Coding Guidelines][authoring-guidelines-general-coding]** because it must be maintained if changes are done to the tests, unlike other submitted solutions. Always think about the future maintainers and translators.
```

Reply:
I wanted users to realize that this snippet poses a maintenance cost, and not that it's just "write once and forget". I have an impression that users do not realize that sloppiness here can cause some unnecessary struggle to maintainers. But indeed I could be too wordy, because I often am :)